docs/features/config/replace-configuration-tokens-with-isecretprovider.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Replace configuration tokens with ISecretProvider"
+layout: default
+---
+
+The `Arcus.WebApi.Security` package provides a mechanism to use your own `ISecretProvider` implementation when building your configuration for your web application.
+
+### Usage
+
+When building your `IConfiguration`, you can call the extension `.AddAzureKeyVault` to pass in your version of the `ISecretProvider` instead on using a specific key vault store name of client.
+
+```csharp
+ISecretProvider yourSecretProvider = ...
```

Comment:
Thinking out loud, would it make sense to show how you can create a secret provider in case folks are new to this?

```suggestion
var vaultAuthenticator = new ManagedServiceIdentityAuthenticator();
var vaultConfiguration = new KeyVaultConfiguration(keyVaultUri);
var yourSecretProvider = new KeyVaultSecretProvider(vaultAuthenticator, vaultConfiguration);
```

Reply:
Yes, makes sense.